### DISCUSSION TITLE
有料コンテンツの販売再開予定について

### DISCUSSION THREAD

**Discussion by I-Tokimoto** [jellywareDev/githubTest#26] [学習コンテンツに関する質問]
現在、有料コンテンツが「販売停止中」となっておりますが、いつから販売が再開されるのでしょうか？

下記の「実践ディープラーニングCNN画像分類（2022/11/05更新）」を購入したいと思っております。
https://monomonotech.jp/learning/contents_trial/deeplearning_cnn/deeplearning_cnn/index.html#0

**Comment by jellyware** [comment]
お問い合わせありがとうございます。
申し訳ございませんが、「実践ディープラーニングCNN画像分類」につきまして、再開できるかを含め検討中です。
目途が立ち次第、ご連絡致します。

**Reply by 7jug** [reply]
同じ文脈ですが、ディープラーニング自動運転アルパカーのほうはいかがでしょう。同じく購入希望です。

**Reply by jellyware** [reply]
お問い合わせありがとうございます。
申し訳ございませんが、「ディープラーニング自動運転アルパカー」につきましても、再開できるかを含め検討中です。
目途が立ち次第、ご連絡致します。
